OZON.ru.js

```diff
@@ -406,24 +131,18 @@ var testCases = [
 						"creatorType": "author"
 					}
 				],
-				"notes": [],
-				"tags": [],
-				"seeAlso": [],
-				"attachments": [
-					{
-						"title": "OZON.ru Link",
-						"mimeType": "text/html",
-						"snapshot": false
-					}
-				],
-				"title": "Язык программирования C++",
-				"language": "ru",
-				"publisher": "Бином, Невский Диалект",
-				"ISBN": "5-7940-0031-7, 5-7989-0127-0, 0-201-88954-4",
-				"date": "1999-1-1",
-				"numPages": "991",
+				"date": "1999",
+				"ISBN": "9785794000313",
```

Comment:
They are, this is just output of `ZU.cleanISBN` - the first of them. There's no mechanism to store multiple ISBNs for one book, and as far as I know ISBN in Zotero is normally stored as a single integer, so just copying the text contents seemed inappropriate.